app/main.py

```diff
@@ -1,6 +1,37 @@
+import os
+
+import requests
+
+
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script. If it's not set, the API request will fail. Consider adding a check to handle the case where `API_KEY` is `None`.